designer/server/src/test/scala/pl/touk/nussknacker/ui/process/deployment/TestDeploymentServiceFactory.scala

```diff
@@ -129,24 +130,34 @@ class TestDeploymentServiceFactory(dbRef: DbRef) {
         dbioRunner
       )
 
-    val validator = ProcessTestData.testProcessValidator(validator = ProcessValidator.default(modelData))
+    val fragmentResolver = new FragmentResolver(newFragmentRepository(dbRef))
     val deploymentService = new DeploymentService(
```

Comment:
Looking at those changeses I think that the way we create the DeploymentService in the tests should be more closely associated with how we create it in the production code. Maybe by some create/factory methods, that we could also use in tests. But that's something for another time, not in this hotfix PR.

Reply:
You are right, I did some changes in this direction in previous changes and in this change as well. There is a lot of work to do in this area.